csharp/sources/Valkey.Glide/GlideClient.cs

```diff
@@ -72,4 +72,13 @@ public static async Task<GlideClient> CreateClient(StandaloneClientConfiguration
 
     public async Task<string> Info(InfoOptions.Section[] sections)
         => await Command(Request.Info(sections));
+
+    public async Task<TimeSpan> PingAsync(CommandFlags flags = CommandFlags.None)
+        => await Command(Request.PingAsync(flags));
```

Comment:
Please rename the request (and echo too)
```suggestion
        => await Command(Request.Ping(flags));
```

Reply:
Are we straying away from the SER naming for these? In that case do we still want to return a `TimeSpan` like SER or do we want to return `OK` like we do for GLIDE?